third_party/ameba/mbedtls.cmake

```diff
@@ -2,7 +2,7 @@ cmake_minimum_required(VERSION 3.6)
 
 project(mbedtls)
 
-set(dir "${sdk_root}/component/common/network/ssl/mbedtls-matter")
+set(dir "${sdk_root}/component/common/application/matter/mbedtls")
```

Comment:
does this depend on some updated SDK? We get build failures it seems.

Reply:
@andy31415 , yes there is a docker update #23759 that needs to be merged first, but that PR also has some merge conflict, we will fix it

Reply:
Ameba build is still failing. I merged with master to see if that makes it work better.